bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractComponentImpl.java

```diff
@@ -129,7 +129,8 @@ private void init() {
         if (currentPage != null && resource != null && request.getAttribute(REQ_ATTR_EMBEDDED_ADAPTIVEFORM) != null) {
             if (!GuideWCMUtils.isForms(getCurrentPage().getPath())) {
                 PageManager pageManager = currentPage.getPageManager();
-                Page resourcePage = pageManager.getContainingPage(resource);
+                String pagePath = (String) request.getAttribute(REQ_ATTR_EMBEDDED_ADAPTIVEFORM);
```

Comment:
What if is this attribute is not set ?

Reply:
In line 129 there is already a condition in place to check if this attribute is not null. Cypress test covers this scenario, let me see about the unit test.